js/samples/04.ai.d.chainedActions.listBot/src/prompts/summarize/skprompt.txt

```diff
@@ -1,3 +1,6 @@
+The following is a conversation with an AI assistant. 
+The assistant can manage lists of items.
```

Comment:
Do these two lines change the expected behavior? It seems like this prompt is used solely to create an adaptive card.

Reply:
At the moment, these two lines are the bare minimum for making the model conversational and getting the adaptive card as a response. It can be extended if you have other suggestions.